### ISSUE TITLE
Run inside a FreeBSD jail?

### ISSUE THREAD

**Issue by kpetrilli** [Kozea/Radicale#1093]
Hi, I am trying to set this up from within a FreeBSD 12.1 jail. When I start the Radicale service up, I encounter this problem:
```
[2020-08-18 19:06:46 +0200] [10429] [INFO] Loaded default config
[2020-08-18 19:06:46 +0200] [10429] [INFO] Loaded config file '/usr/local/etc/radicale/config'
[2020-08-18 19:06:46 +0200] [10429] [INFO] Starting Radicale
[2020-08-18 19:06:46 +0200] [10429] [INFO] auth type is 'radicale.auth.none'
[2020-08-18 19:06:46 +0200] [10429] [INFO] storage type is 'radicale.storage.multifilesystem'
[2020-08-18 19:06:46 +0200] [10429] [INFO] rights type is 'radicale.rights.owner_only'
[2020-08-18 19:06:46 +0200] [10429] [INFO] web type is 'radicale.web.internal'
[2020-08-18 19:06:46 +0200] [10429] [INFO] Listening on '[0.0.0.0]:8080'
[2020-08-18 19:06:46 +0200] [10429] [CRITICAL] An exception occurred during server startup: Failed to start server '[0.0.0.0]::5232': [Errno 43] Protocol not supported
```
I couldn't find anything about **Errno 43** online, and the program actually works outside of the jail (from within the host system). I guess I am either missing a dependency or there is some other problem.

**Comment by Unrud** [comment]
> I couldn't find anything about **Errno 43** online

EPROTONOSUPPORT

> I guess I am either missing a dependency or there is some other problem.

It's a bug in Radicale. The errno code is missing here:

https://github.com/Kozea/Radicale/blob/f950ce98ab07d6095b5f603cff364725b36b485f/radicale/server.py#L226-L243

I will create a fix.

**Comment by Unrud** [comment]
Fixed in version 3.0.4

**Comment by kpetrilli** [comment]
That's great, thanks
